### PR TITLE
Feature/ethtool/5.16

### PR DESCRIPTION
**Pull request by ParianGol**

### CODE REVIEW

File: analysed-packages/ethtool/version-5.16/ethtool-5.16.spdx.yaml
Comment (OliverFendt) [inline analysed-packages/ethtool/version-5.16/ethtool-5.16.spdx.yaml:1]: Please resolve in this file my comments in the disclosure file and tag value file, too

File: analysed-packages/ethtool/version-5.16/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+## Reviewers
+
+The information was reviewed by:
+
+* Dorothea Kraus
+* Oliver Fendt
+
+## Comment
+
+This package is dual licensed under GPL-2.0-only OR BSD-2-Clause. To facilitate automatic use, a default license choice of BSD-2-Clause is denoted in the Acknowledgment section of the OSS disclosure document. This shall not restrict the freedom of users to choose either GPL-2.0-only OR BSD-2-Clause. For convenience all license texts are provided.
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/ethtool/version-5.16/README.md]: https://github.com/Open-Source-Compliance/package-analysis/pull/111/commits/974b690951c6b6667a1d6aa11d0fe96325b572e8

Thank you!

File: analysed-packages/ethtool/version-5.16/ethtool-5.16-OSS-disclosure.txt [hunk 2]
```diff
@@ -0,0 +1,1777 @@
...
+Copyright (C) 2008,2009 Intel Corporation Author: Patrick Ohly
+<patrick.ohly@intel.com>
+Copyright (C) 2008 Sun Microsystems, Inc. All rights reserved.
+Copyright (C) 2007-2009 STMicroelectronics Ltd
+Copyright (C) 2006-2017 Free Software Foundation, Inc.
+Copyright (C) 2004-2017 Free Software Foundation, Inc.
+Copyright (C) 2004, 2006 Stephen Hemminger <shemminger@osdl.org>
+Copyright (C) 2004 Intracom S.A. Pantelis Antoniou <panto@intracom.gr>
+Copyright (C) 2003-2017 Free Software Foundation, Inc.
+Copyright (C) 2003 Advanced Micro Devices Inc. */ include <stdio.h> include
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/ethtool/version-5.16/ethtool-5.16-OSS-disclosure.txt]: https://github.com/Open-Source-Compliance/package-analysis/pull/111/commits/33347e7b67fc6facf12ca9aabf992768d1a92868 
Done, thank you!

File: analysed-packages/ethtool/version-5.16/ethtool-5.16-SPDX2TV.spdx [hunk 3]
```diff
@@ -0,0 +1,3323 @@
...
+PackageFileName: ethtool-5.16.tar.gz
+SPDXID: SPDXRef-upload214
+PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: true
+PackageVerificationCode: be67b8082c6bd454da9779f0b7188db0e7aa7309
+PackageChecksum: SHA1: 678e62a9d7a26016aa1c06b09b44858eedb79d99
+PackageChecksum: SHA256: 4160fa127d75d5a6ebd5e871118486ac730dea2f28b2f1ad40fd1ed3d0d369e0
+PackageChecksum: MD5: bd960a9b141c8e6448d4e847cd945bc1
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: LicenseRef-GPL-2.0-or-later
```
[the remark this answers is not in the document]
Comment (ParianGol) [reply analysed-packages/ethtool/version-5.16/ethtool-5.16-SPDX2TV.spdx]: Done, thank you!
https://github.com/Open-Source-Compliance/package-analysis/pull/111/commits/8b929f1b9b59732cccd44d29a7db64a9378f4dd5

**Review by OliverFendt — APPROVED** [review]
@ParianGol awesome, thank you very much.

**Pull request merged**